robotframework-ls/src/robotframework_ls/impl/find_definition.py

```diff
@@ -237,6 +237,7 @@ def find_definition(completion_context: ICompletionContext) -> Sequence[IDefinit
                 completion_context.token_value_resolving_variables(token),
                 create=True,
                 current_doc_uri=completion_context.doc.uri,
+                remote=ast_utils.is_remote_library_node(token_info.node),
```

Comment:
In this case the remote library name is not being changed (probably to get to the actual `Remote` library in a find definition).

It'd be nice to have a test for this case in: 
`robotframework-ls/tests/robotframework_ls_tests/test_find_definition.py`

Reply:
It actually is :) It is just not in the diff of this file. `token` is acquired by calling `ast_utils.get_library_import_name_token` that returns what we need further.

Reply:
Humm, ok, just saw that... I'm not sure that's ideal. 

What happens in the case where `token.value = get_library_name_from_node(ast)` but `get_library_name_from_node(ast)` returns `None`?

-- the main things here would be the behavior for `code analysis`/`code-completion`/`find definition`.

i.e.: the user created a library with `Remote` but still hasn't specified any arguments.